sdk/core/Azure.Core.TestFramework/src/TestProxy.cs

```diff
@@ -70,7 +70,7 @@ private TestProxy(string proxyPath, bool debugMode = false)
 
             ProcessStartInfo testProxyProcessInfo = new ProcessStartInfo(
                 s_dotNetExe,
-                $"\"{proxyPath}\" --storage-location=\"{TestEnvironment.RepositoryRoot}\"")
+                $"\"{proxyPath}\" start -u --storage-location=\"{TestEnvironment.RepositoryRoot}\"")
```

Comment:
what is this change for?

Reply:
When I updated the proxy to output to stderr it broke a bunch of your tests. I've been publishing a version for ya'll that doesn't output to stderr for compatibility reasons (no time to investigate failures).